scripts/script.js

```diff
@@ -36,11 +36,27 @@ const db = getFirestore(app);
 //DECLARE scoreText global array
 //DECLARE answersClickable global variable to true
 
-//on page load, onclick difficulty and onclick try again from end page
+//Onclick difficulty and onclick try again from end page
 function toggleLandingPage() {
-  //TOGGLE div class to show or hide the landing page
+  document.getElementById("landing-page").classList.toggle("hidden-overlay");
 }
 
+setTimeout(function(){ 
+  document.querySelector(".landing-btn").style.pointerEvents = 'auto';
+  document.querySelector(".landing-btn--alternate").style.pointerEvents = 'auto';
+ }, 35000);
+
+//Hide the landing page overlay and start the game when the easy or hard button is clicked
+document.getElementById("easy-btn").addEventListener("click", function() {
+  toggleLandingPage();
+  startGame(easy);
```

Comment:
I think it would be good to set the global variable "currentDifficulty" to "easy" here because its used afterwards. You could then insert the variable as the argument.